_posts/2015-04-02-welcome-to-sittercity-tech.markdown

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title:  "Welcome to Sittercity Technology!"
+date:   2015-04-02 09:00:00
+categories: sittercity
+---
+
+Welcome! This here is our new blog! As is fitting of a new journal let's start ourselves off with an introduction to Sittercity Tech.
```

Comment:
Aaaaah  I didn't get it but let's see what other's say! I'm still learning this folksy stuff